annlite/storage/kv.py

```diff
@@ -111,8 +111,13 @@ def close(self):
                 '`DocStorage` had been closed already, will skip this close operation.'
             )
             return
-        self._db.flush(wait=True)
-        self._db.close()
+        try:
```

Comment:
does this mean that the error is caused by not closing RocksDB correctly?

Reply:
In some cases, the data file is forcefully deleted by another program or `rm -rf`.

Reply:
oh, i see, the uploading brokes our tests. https://github.com/jina-ai/annlite/actions/runs/4507514031/jobs/7935366275?pr=222 @jemmyshin Do you have any clues about it?